core/src/test/scala/integration/kafka/admin/TopicCommandIntegrationTest.scala

```diff
@@ -845,24 +812,19 @@ class TopicCommandIntegrationTest extends KafkaServerTestHarness with Logging wi
       Set(new TopicPartition(testTopicName, 0)).asJava
     )
 
-    adminClient.createTopics(
-      Collections.singletonList(new NewTopic(testTopicName, 1, 1.toShort))
-    ).all().get()
-    waitForTopicCreated(testTopicName)
+    TestUtils.createTopicWithAdmin(adminClient, testTopicName, brokers, 1, 1)
 
     val output = TestUtils.grabConsoleOutput(
       topicService.describeTopic(new TopicCommandOptions(Array("--topic", testTopicName))))
-    val rows = output.split("\n")
+    val rows = output.split(System.lineSeparator())
     assertEquals(2, rows.size)
```

Comment:
please print output here as well. It helps in debugging when this assertion fails.

Reply:
Done.